### PR TITLE
fix for #121. Dom element didn't wait for timeout

### PR DESCRIPTION
**Pull request by vefimofff**

### CODE REVIEW

File: src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyFluentWait.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.wiley.autotest.selenium.elements.upgrade;
 
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
```
Comment (estuold) [inline src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyFluentWait.java]: Incorrect import, use NoSuchElementException

File: src/test/java/com/wiley/autotest/framework/pages/TestDomElementPage.java [hunk 2]
```diff
@@ -20,6 +22,13 @@ public TestDomElementPage checkDomElementBecomeVisible() {
         return this;
     }
 
+    public TestDomElementPage checkDomElementAppearAfterTimeout() {
```
Comment (estuold) [inline src/test/java/com/wiley/autotest/framework/pages/TestDomElementPage.java]: I don't see any timeouts and checks for timeout in method.

File: src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyFluentWait.java (hunk 1 above)
Comment (vefimofff) [reply src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyFluentWait.java]: it was unused. removed

File: src/test/java/com/wiley/autotest/framework/pages/TestDomElementPage.java (hunk 2 above)
Comment (vefimofff) [reply src/test/java/com/wiley/autotest/framework/pages/TestDomElementPage.java]: the "logic" is in the HTML page... (an element is attached to dom after 6 seconds). Will add a comment to the method - unless you have some suggestions on how to better name it?

**Pull request merged**